script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -612,12 +613,13 @@ The world changes.
 [sha:36d2ef2e347c01132b1c607e4bc81fc9f0abfd14]{
 -- Page 344, Offset 32621.
 -- 見知らぬ風景に切り替わる。
-It switches to an unfamiliar scene.
+It transforms to an unfamiliar scene.
 }
 [sha:6479304defc2e9ca2e4349efb7675a7ec3a86c8b]{
 -- Page 344, Offset 32622.
 -- その外側、彼女の宇宙の外にいる俺ですら、見えない圧力で全身が締め付けられる。
-Even from outside her space, I feel an invisible pressure tightening on my entire body.
+// space here is 宇宙 but dont want to mess around with it too much because well...... nasu terms
+Even I, standing outside of her space, feel an invisible pressure tightening on my entire body.
```

Comment:
Even I, standing outside of her independent universe, feel an invisible pressure over my entire body.